code/modules/cargo/packs.dm

```diff
@@ -1507,6 +1507,28 @@
 					/obj/item/stack/tile/carpet/black/fifty)
 	crate_name = "premium carpet crate"
 
+/datum/supply_pack/service/carpet_exotic
+	name = "Exotic Carpet Crate"
+	desc = "Plasteel floor tiles getting on your nerves? These stacks of extra soft carpet will tie any room together."
```

Comment:
1. I would change the description to differ from the premium carpet crate (line 1502)
2. It might be worth adding what the crate contains quantitatively while you're in here, many of the other crates already do. ("...Contains 100 tiles of 8 different varieties of carpet..." or something)

Reply:
done and done, good suggestion